scripts/lint-packages.sh

```diff
@@ -129,12 +129,12 @@ check_indentation() {
 }
 
 {
-	# We'll need the termux/termux-packages master@HEAD commit as a base commit when running the version check.
+	# We'll need the termux-packages master@HEAD commit as a base commit when running the version check.
```

Comment:
Since we're decoupling this from a hardcoded URL, let's just refer to it as `origin`, since that's the remote name.
```suggestion
	# We'll need the origin/master HEAD commit as a base commit when running the version check.
```